### PR TITLE
Fix warnings and implement `cargo apk run`

### PR DESCRIPTION
**Pull request by yoanlcq**
Greetings,

Hopefully the title says all, but I'd like to give some details : 

I claim to have "fixed" warnings, but I am worried that there is an actual reason for keeping them which I don't know about (because I think they would have been fixed before otherwise). Or could it be a difference between rustc versions ? FWIW, I'm on 1.20 nightly.  
Also most warnings were caused by parts claimed to be generated by `android-rs-bindgen` (in `injected_glue`). Perhaps `android-rs-bindgen` would need a fix too, but I dunno.  

Anyway I've added `#![deny(warnings)]` so this doesn't happen again.

And well, `cargo apk run` is probably a feature we all want. :)

I'm not sure about the "version bump" in my latest commit though.

### CODE REVIEW

File: cargo-apk/src/main.rs [hunk 1]
```diff
@@ -1,3 +1,5 @@
+#![deny(warnings)]
```
Comment (tomaka) [inline cargo-apk/src/main.rs]: Please remove the deny(warnings). New versions of Rust tend to introduce new warnings, and I don't want the break to stop compiling by itself.

**Comment by tomaka** [comment]
> Perhaps android-rs-bindgen would need a fix too, but I dunno.

These bindings were generates decades ago, so they should maybe be regenerated eventually.

File: cargo-apk/src/main.rs (hunk 1 above)
Comment (yoanlcq) [inline cargo-apk/src/main.rs]: Will do.

**Comment by yoanlcq** [comment]
By the way, CI should probably also test `cargo apk run`.

**Pull request merged**